skimage/filters/_gaussian.py

```diff
@@ -96,6 +98,11 @@ def gaussian(image, sigma=1, output=None, mode='nearest', cval=0,
     >>> filtered_img = gaussian(image, sigma=1, multichannel=True)
 
     """
+    if preserve_range is None:
+        msg = ('The default value of preserve_range will change '
+               'to "True" in version 1.0')
```

Comment:
```suggestion
               'to True in version 1.0')
```
@scikit-image/core can you please have a look and say if you agree with this API change?